Flutter/assistview/getting-started.md

```diff
@@ -92,8 +194,42 @@ When the send button is clicked, the [`AssistActionButton.onPressed`] callback i
 
 {% tabs %}
 {% highlight dart %}
+
+  late List<AssistMessage> _messages;
+
+  void _generativeResponse(String data) async {
+    final String response = await _getAIResponse(data);
+    setState(() {
+      _messages.add(AssistMessage.response(data: response));
+    });
+  }
+
+  Future<String> _getAIResponse(String data) async {
+    String response = '';
+    // Connect with your preferred AI to generate a response to the request.
+    return response;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SfAIAssistView(
+        messages: _messages,
+        actionButton: AssistActionButton(
+          onPressed: (String data) {
+            setState(() {
+              _messages.add(AssistMessage.request(data: data));
+            });
+            _generativeResponse(data);
+          },
+        ),
+      ),
+    );
+  }
 	
 {% endhighlight %}
 {% endtabs %}
 
+![Action button chat](images/getting-started/actionbutton-chat.png)
```

Comment:
Here add demo as gif

![Scrollable placeholder](https://github.com/user-attachments/assets/3bc42cf1-8e77-4bcf-88f1-953eac40a329)

Like this but with your above codes

Ask single question alone